src/components/DocumentationTopic/DocumentationNav/Hierarchy.vue

```diff
@@ -91,18 +99,54 @@ export default {
       default: () => [],
     },
   },
+  data() {
+    return {
+      windowWidth: window.innerWidth,
+    };
+  },
+  mounted() {
+    // start tracking the window size
+    const cb = throttle(() => { this.windowWidth = window.innerWidth; }, 150);
```

Comment:
Can you explain the 150 limit? Why is it need?

Reply:
Its a general good practice to throttle window resize listeners, as they can be very noisy. In this case we dont need pixel-perfect positioning, just a rough estimate so we can push the hierarchy items into collapsing.